indico/modules/users/controllers.py

```diff
@@ -639,3 +639,16 @@ class RHUserSearchInfo(RHProtected):
     def _process(self):
         external_users_available = any(auth.supports_search for auth in multipass.identity_providers.itervalues())
         return jsonify(external_users_available=external_users_available)
+
+
+class RHUserBlock(RHUserBase):
+    def _block(self, blocked):
+        self.user.is_blocked = blocked
+        flash(_('The user has been {}.'.format('blocked' if blocked else 'unblocked')), 'success')
```

Comment:
This is not how i18n works ;)

- you need to have a static string inside `_(...)` for extraction to work
- concatenating strings makes translation hard to impossible (for example, blocked/unblocked are not translated right now, and even if you wrapped those in `_()`, it would only work for languages where the word is at the end of the sentence)

So better to use two separate (fully translatable) sentences.

Reply:
That makes sense, I forgot about languages where the word could change place.